shared/.flowconfig

```diff
@@ -44,6 +44,8 @@ sketchy-null-bool=off
 sketchy-null-string=off
 sketchy-null-mixed=off
 sketchy-null-number=off
+unclear-type=off
+untyped-import=off
```

Comment:
this is kinda useful to see, but mostly we know about this. coverage will also reveal this kind of thing